[runtime error] deerflow cannot start dev via make on windows

### Problem summary

after make install ,running make dev I have this error :process_begin: CreateProcess(NULL, env bash C:\ai_env\deer-flow\scripts\serve.sh --dev, ...) failed. make (e=2): 系统找不到指定的文件。 mingw32-make: *** [Makefile:93: dev] Error 2

### Expected behavior

C:\MinGW\bin\mingw32-make.exe dev can start the server and web ui

### Actual behavior

<img width="1252" height="268" alt="Image" src="https://github.com/user-attachments/assets/8df93c7a-85b5-4217-97f4-265611fbadaf" />

### Operating system

Windows

### Platform details

amd

### Python version

3.13.10

### Node.js version

v22.16.0

### pnpm version

10.29.3

### uv version

uv 0.10.9 (f675560f3 2026-03-06)

### How are you running DeerFlow?

Local (make dev)

### Reproduction steps

C:\MinGW\bin\mingw32-make.exe check (pass without error)

no optional make build up

C:\MinGW\bin\mingw32-make.exe dev  failed



### Relevant logs


<img width="1252" height="268" alt="Image" src="https://github.com/user-attachments/assets/8df93c7a-85b5-4217-97f4-265611fbadaf" />


### Git state

_No response_

### Additional context

_No response_

--- a/Makefile
+++ b/Makefile
@@ -3,13 +3,11 @@
 .PHONY: help config config-upgrade check install dev dev-daemon start stop up down clean docker-init docker-start docker-stop docker-logs docker-logs-frontend docker-logs-gateway
 
 PYTHON ?= python
+BASH ?= bash
 
 # Detect OS for Windows compatibility
 ifeq ($(OS),Windows_NT)
     SHELL := cmd.exe
-    BASH := bash
-else
-    BASH := bash
 endif
 
 help:

